seed/python-sdk/alias/src/seed/core/file_types.py

```diff
@@ -0,0 +1,32 @@
+# This file was auto-generated by Fern from our API Definition.
+
+import typing
+
+# File typing inspired by the flexibility of types within the httpx library
+# https://github.com/encode/httpx/blob/master/httpx/_types.py
+FileContent = typing.Union[typing.IO[bytes], bytes, str]
+FileTypes = typing.Union[
+    # file (or bytes)
+    FileContent,
+    # (filename, file (or bytes))
+    typing.Tuple[typing.Optional[str], FileContent],
+    # (filename, file (or bytes), content_type)
+    typing.Tuple[typing.Optional[str], FileContent, typing.Optional[str]],
+    # (filename, file (or bytes), content_type, headers)
+    typing.Tuple[typing.Optional[str], FileContent, typing.Optional[str], typing.Mapping[str, str]],
+]
+
+# The format we use is a list of tuples, where the first element is the name of the file and the second is the file object
+# Typically HTTPX wants a dict, but to be able to send lists of files, you have to use the list approach (which also works for non-lists)
+# https://github.com/encode/httpx/pull/1032
```

Comment:
nit: let's reduce the length of the docs here / use a multiline string (whatever makes it look prettier)
<img width="993" alt="Screenshot 2024-02-19 at 11 24 35 AM" src="https://github.com/fern-api/fern/assets/10870189/b139b9da-1a81-4cd7-8f00-80dcb33dea34">